.github/workflows/my-ci-pipeline.yml

```diff
@@ -55,7 +55,38 @@ jobs:
     - name: Push Docker Prod Image v2
       run: docker push rsalhan/m8-todo-app:prod
 
-    - name: curl Command
-      env:
-        webhook_url: ${{ secrets.WEBHOOK_URL }}
-      run: curl --fail -dH -X POST $webhook_url
+  terraform:
+    name: Terraform Init/Plan/Apply
+    runs-on: ubuntu-latest
+    needs: job-two
```

Comment:
This might be a reminder that our job names could be a bit clearer! 😄

Reply:
Cheers Jack, noted, laziness on my part tbh when I first setup that second job.